app/repository_models/person.rb

```diff
@@ -123,6 +123,7 @@ def user
   def to_solr(solr_doc={}, opts={})
     super(solr_doc, opts)
     Solrizer.set_field(solr_doc, 'generic_type', 'Person', :facetable)
+    Solrizer.set_field(solr_doc, 'sort_title', "#{self.last_name}, #{self.first_name}", :stored_sortable)
```

Comment:
There's a method that gives the inverted name. It's technically on the user model, so you'd have to hop to that, but it handles cases where the name may not be set, or where a field is blank:

https://github.com/uclibs/curate/blob/develop/app/models/curate/user_behavior/base.rb#L54-L58